thirdparty/faiss/faiss/IndexAdditiveQuantizer.cpp

```diff
@@ -174,6 +175,7 @@ void search_with_LUT(
         if (!is_IP) { // the LUT function returns ||y||^2 - 2 * <x, y>, need to
                       // add ||x||^2
             bias = fvec_norm_L2sqr(xq + q * d, d);
+            bias = (bias == 0.0 ? 1.0 : bias);
```

Comment:
no, because it uses norm for distance = x^2 - 2xy + y^2 calculation, not for a cosine-related code